code/modules/projectiles/gun_system.dm

```diff
@@ -413,6 +413,8 @@
 		O.emp_act(severity)
 
 /obj/item/weapon/gun/equipped(mob/user, slot)
+	if(ishandslot(slot) || slot == SLOT_BACK || slot == SLOT_BELT || slot == SLOT_S_STORE)	//add more if needed
```

Comment:
Are there slots you'd equip it to without wanting it to be opaque? This just seems like the list of places guns fit.

Reply:
I've set it to the places where you would equip guns because these are where guns are tac-reloadable.
In testing, if you click drag a magazine on a gun, but you miss the gun sprite, you will fail your tac reload. 
You can tac reload a gun on back/belt/suit storage or hands, so I'm making it opaque to match those slots.

It also just makes it easier to click on the gun sprite to draw out of those slots.